htdocs/core/triggers/interface_70_modpropale_MailPropalSigne.class.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/* Copyright (C) 2022      Brice
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ *  \file       htdocs/core/triggers/interface_70_modpropale.MailPropalSigne.class.php
```

Comment:
Rename your trigger interface_50_modPropale_Mail.class.php and use the trigger code PROPAL_CLOSE_SIGNED
and send email when $object->context contains the key closedfromonlinesignature